consensus/parlia/abi.go

```diff
@@ -1,1525 +1,3564 @@
 package parlia
 
+const validatorSetABIBeforeBoneh = `
+[
```

Comment:
The `abi` here needs to be regenerated according to the latest version.  Please confirm.

Reply:
ok